app/src/main/res/values-uk/strings.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- PC view menu entries -->
+    <string name="pcview_menu_app_list">Подивитися список ігор</string>
+    <string name="pcview_menu_pair_pc">Створити пару з ПК</string>
+    <string name="pcview_menu_unpair_pc">Розірвати пару</string>
+    <string name="pcview_menu_send_wol">Надіслати Wake-On-LAN запит</string>
+    <string name="pcview_menu_delete_pc">Видалити ПК</string>
+
+    <!-- Pair messages -->
+    <string name="pairing">Створення пари…</string>
+    <string name="pair_pc_offline">Комп'ютер вимкнений або знаходиться не в мережі</string>
```

Comment:
These `'` characters must be escaped in these XML files. You should use `\'` instead of `'` to avoid build errors like `AAPT: error: unescaped apostrophe in string`

https://travis-ci.org/github/moonlight-stream/moonlight-android/builds/705048446